Controls/Grid/Grid.UWP/Model/GridModel.Columns.cs

```diff
@@ -90,6 +93,37 @@ internal static void UpdateFilterMemberAccess(IFieldInfoData data, IEnumerable<F
             }
         }
 
+        internal static PropertyInfoFieldInfo InitializePropertyInfo(string nestedPropertyPath, Type type)
+        {
+            string[] splitNestedProperty = nestedPropertyPath.Split(new char[] { NestedPropertySeparator });
```

Comment:
Can use split(".");

Reply:
For now we can't because we are suing a lower version of the .NETCore package where Split by string is not available. I hope soon I will be able to update the package.